javascript/new-src/src/app.js

```diff
@@ -20,6 +20,15 @@
 /* global angular */
 var ola = angular.module('olaApp', ['ngRoute']);
 
+// AngularJS 1.6 now uses an '!' as the default hashPrefix, breaking
+// our routes. Following lines revert the default to the previous empty
+// string.
```

Comment:
I think we could also add a note saying given this webpage is only used internally, there is no benefit in the web crawler changes.

We could possibly add a TODO to update our URLs to remove this change, but maybe that's not necessary if we're rewriting to a new framework in future?

Reply:
I agree that this might be useful but since AngularJS (= 1.x) is a dead end, we won't bother for now